test/empty-options.test.js

```diff
@@ -121,7 +123,7 @@ describe('when applied with no options', () => {
           }], () => {
             expect(compilation.errors.length).toBe(1);
             expect(compilation.errors[0]).toBeInstanceOf(Error);
-            expect(compilation.errors[0].message).toEqual(expect.stringContaining('TypeError'));
```

Comment:
Just making note of this for when #80 is rebased on top of this. This should remain a TypeError - but I think the options schema validation should cover this.